src/components/IncidentInfo/IncidentInfo.js

```diff
@@ -22,11 +33,15 @@ const IncidentInfo = ({ id }) => {
         {console.log(incident)}
         <p data-cy='info-name' className='info info-name'>{incident.name}</p>
         <p data-cy='info-type' className='info type'><span className='info-type'>{getIconByType(incident.incident_type)}</span>{incident.incident_type}</p>
-        <a href={mapURL} target="_blank" rel="noopener noreferrer" className='map-btn'>Click to get Map to Headquarters</a>
+        <div className="info-button-container">
+          <a href={mapURL} target="_blank" rel="noopener noreferrer" className='map-btn'>Click to get Map to Headquarters</a>
+          {!incident.close_date && <button className="end-button" onClick={e => handleSubmit(e)}>Declare Incident Over</button>}
+        </div>
         <p data-cy='info-location' className='info info-location'>Located at: {incident.location} in {incident.city}, {incident.state}</p>
         <p data-cy='info-desc' className='info info-desc'>Summary: {incident.description}</p>
         <p data-cy='info-start-date' className='info info-start-date'>Declaration: {formatDate(incident.start_date)}</p>
-        <p data-cy='info-end-date' className='info info-end-date'>Resolution: {formatDate(incident.close_date)}</p>
+        {console.log(incident.close_date)}
+        {incident.close_date && <p data-cy='info-end-date' className='info info-end-date'>Resolution: {formatDate(incident.close_date)}</p>}
         <div data-cy='info-instructions' className='info info-instructions'>Reminders & Instructions: {getInstructions(incident.incident_type)}</div>
```

Comment:
Something else we might do here:
`<p data-cy='info-end-date' className='info info-end-date'>Resolution: {incident.active ? 'Ongoing' : formatDate(incident.close_date)}</p>`

Reply:
or Ongoing: formatDate(Date.now)?